recipes/gtsam/all/test_package/conanfile.py

```diff
@@ -11,6 +12,5 @@ def build(self):
         cmake.build()
 
     def test(self):
-        if not tools.cross_building(self.settings):
```

Comment:
If we don't check for crosscompilation it will fail each time we try to execute a binary compiled for a different architecture.

Reply:
Yup my bad, that line should have stayed there.